streamer/src/nonblocking/swqos.rs

```diff
@@ -259,9 +205,6 @@ impl SwQos {
                 update_open_connections_stat(&self.stats, &connection_table_l);
                 drop(connection_table_l);
 
-                if let Ok(receive_window) = receive_window {
-                    connection.set_receive_window(receive_window);
-                }
                 connection.set_max_concurrent_uni_streams(max_uni_streams);
```

Comment:
Why not apply RTT-based scaling like we introduce in simple_qos? In this case, we should also ensure that senders don’t receive less than before -- at least until we have auto-tuning, and maybe even afterward unless these defaults turn out to be inadequate.